jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/QuotedQualityCSV.java

```diff
@@ -148,15 +149,8 @@ protected void parsedValueAndParams(StringBuilder buffer)
     @Override
     protected void parsedValue(StringBuilder buffer)
     {
-        super.parsedValue(buffer);
-
         // We have to convert to String anyway for QualityValue below.
         String value = buffer.toString();
-        // Ensure we don't have whitespace or control only value
-        if (StringUtil.isBlank(value))
-            return;
```

Comment:
Why get rid of this?
Is this done somewhere else now?

Reply:
OK